swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/AbstractResourceTypeHandler.kt

```diff
@@ -516,78 +469,75 @@ abstract class AbstractResourceTypeHandler<T : Resource>(
     }
   }
 
-  private fun doDelete(workConfiguration: WorkConfiguration, postClean: () -> Unit) {
+  private fun doDelete(workConfiguration: WorkConfiguration) {
     exclusionCounters[Action.DELETE] = AtomicInteger(0)
     val candidateCounter = AtomicInteger(0)
     val markedResources = mutableListOf<MarkedResource>()
-    try {
-      val currentMarkedResourcesToDelete = resourceRepository.getMarkedResourcesToDelete()
-        .filter {
-          it.notificationInfo != null && !workConfiguration.dryRun && it.namespace == workConfiguration.namespace
-        }
-        .sortedBy { it.resource.createTs }
 
```

Comment:
Why did you remove the `try / catch` from here (and notify)? Do you think that we won't encounter any errors? Or are they handled someplace else?

Reply:
Removed the `try ... finally` that was there  to ensure the `postAction` callbacks get called.
Those callbacks are no longer needed. There is a try/catch to handle exception when an individual resource causes to throw.

Other exceptions that bubble up are handled by the caller: `ScheduledAgent#process` https://github.com/spinnaker/swabbie/pull/214/files#diff-facdb46f9906b5a7113a4110c7cc1813L146

Reply:
👍